Samples/ExplicitlyGroupedKernels/Program.cs

```diff
@@ -34,8 +34,8 @@ class Program
         /// <param name="dataView">The view pointing to our memory buffer.</param>
         /// <param name="constant">A nice uniform constant.</param>
         static void GroupedKernel(
-            ArrayView<int> dataView,     // A view to a chunk of memory (1D in this case)
```

Comment:
technically I think it can't be done due to conflict with a general rule to remove extra white space (which I found pretty important in general formating)

Reply:
update: we can disable space checking, but i think it will have more cons then pros 😢